package.json

```diff
@@ -44,6 +45,7 @@
     "@semantic-release/git": "^9.0.0",
     "cypress": "^5.1.0",
     "ejs-lint": "^1.1.0",
+    "mocha": "^8.1.3",
```

Comment:
Like I mentioned in the comment I think we should keep this as `jest` for continuity sake. glad you got this in though!